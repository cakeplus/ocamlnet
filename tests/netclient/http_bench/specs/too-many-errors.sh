. helpers.sh

# The client discards the request after the second failed connection.
# So the first GET reports an error, and the second GET succeeds.

start_test_server \
   -line 1 -break \
   -line 1 -break \
   -line 1 -file data/framed
trap "stop_test_server" EXIT
request -get / -run -get / -run
