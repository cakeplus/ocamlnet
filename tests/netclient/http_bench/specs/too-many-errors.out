Http_protocol: Bad_message: Protocol error
22F1A22D055CDBCC183A9FC3FCCA7732
